### PR TITLE
Migrate admin pages to React

### PR DESCRIPTION
**Pull request by lightnet328**
# Overview
There is no major change in appearance or feature, and admin pages were migrated to React.

# Changes

- Migrated admin pages to React
- Introduced partial SPA
  - Installed `react-router`
  - Make transitions within admin pages on one page
  - Remove swig templates & codes that are no longer used
- Partial translation
- Replacement text with the key for translation

# Depends on
#420

### CODE REVIEW

File: client/components/Admin/Navigation.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    }
+  })
+
+  return (
+    <div className="nav nav-pills flex-column">
+      {items.map(
+        ({ key, to, icon, name, exact = false, show = true }) =>
+          show && (
+            <NavLink key={key} className="nav-link" to={to} exact={exact} activeClassName="active">
+              <i className={`mdi ${icon}`} /> {name}
```
Comment (sotarok) [inline client/components/Admin/Navigation.tsx]: 💡  You can write `<Icon name={icon} />` with using Common/Icon components

File: locales/en-US/translation.yml [hunk 2]
```diff
@@ -223,8 +226,113 @@ search:
     results: '{{value}} results'
 
 admin:
+  app:
+    legend: App Settings
+    title: Wikiの名前
```
[the remark this answers is not in the document]
Comment (lightnet328) [reply locales/en-US/translation.yml]: There are too many sentences to translate, so I think PR should be separated.

File: client/components/Admin/AdminPage.tsx [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+  isUploadable: false,
+} as any) as AdminContext)
+
+function useSearchConfig(crowi: Crowi) {
+  const [searchConfigured, setSearchConfigured] = useState(false)
+
+  const fetchSearchConfig = async () => {
+    const { searchConfigured: isConfigured = false } = await crowi.apiGet('/admin')
+    setSearchConfigured(isConfigured)
+  }
```
[the remark this answers is not in the document]
Comment (lightnet328) [reply client/components/Admin/AdminPage.tsx]: Maybe, It is for managing the loading status.

**Comment by sotarok** [comment]
@lightnet328 Can you resolve the conflict?

**Comment by sotarok** [comment]
And the build failed.

**Pull request merged**